bouncer/views.py

```diff
@@ -69,7 +69,12 @@ def annotation(self):
         extension_url = "{uri}#annotations:{id}".format(
             uri=document_uri, id=annotation_id)
 
-        pretty_url = _pretty_url(document_uri)
+        pretty_url = util.make_pretty_url(document_uri)
+
+        if can_reveal_metadata:
+            title = "Hypothesis annotation for {site}".format(site=pretty_url)
+        else:
+            title = "Hypothesis annotation"
```

Comment:
These two strings are already defined in functions (or constants) in the `util` package, why not just use those instead of copying them here?
They're also not translatable at the moment.

In addition to that, there doesn't seem to be any tests for both of these branches, right?